setup.py

```diff
@@ -47,7 +47,7 @@
             "flake8-bugbear",  # flake8 doesn't have a dependency for bugbear plugin
             "flake8>=3.7,<5",
             "isort>=4.3,<5.2.0",
-            "mypy>=0.770,<0.800",
+            "mypy>=0.770,<1.0.0",
```

Comment:
Please keep this line. This line just declares recommended version (not supported versions).